### ISSUE TITLE
[UPGRADE] - EKS 1.22

### ISSUE THREAD

**Issue by kcoleman731** [aws-ia/terraform-aws-eks-blueprints#431]
Add support for Kubernetes v1.22

**Comment by agido-heppe** [comment]
For me, KubeProxy, AWS Load Balancer Controller and EBS CSI Driver didn't work out of the box.

Following adjustments to the charts fixed this for me in EKS v1.22.


 ```
 aws_load_balancer_controller_helm_config = {
    version = "1.4.1"
  }

  amazon_eks_kube_proxy_config = {
    addon_version = "v1.22.6-eksbuild.1"
  }

  amazon_eks_aws_ebs_csi_driver_config = {
    addon_version = "v1.5.3-eksbuild.1"  
  }
```

**Comment by bryantbiggs** [comment]
Closed in #634
